pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -752,6 +523,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 	}
 
+	roles = append(roles, InfraSAs.AllRoles()...)
```

Comment:
make sure we don't define duplicate roles here... that's probably error/panic worthy